site/_docs/reference.md

```diff
@@ -1510,10 +1510,6 @@ Not implemented:
 | {fn TIMESTAMPADD(timeUnit, count, datetime)} | Adds an interval of *count* *timeUnit*s to a datetime
 | {fn TIMESTAMPDIFF(timeUnit, timestamp1, timestamp2)} | Subtracts *timestamp1* from *timestamp2* and returns the result in *timeUnit*s
 
-Not implemented:
-
-* {fn DAYNAME(date)}
-* {fn MONTHNAME(date)}
```

Comment:
JDBC String function` {fn DAYNAME(date)}` and `{fn RIGHT(string, integer)}` not create in` SqlJdbcFunctionCall `and `SqlOperatorBaseTest.testJdbcFn()` not test.

Reply:
hi @ambition119  Thank you very much.